internal/api/openapi.yaml

```
@@ -4271,6 +4305,144 @@ paths:
+                    type: string
+                    pattern: "^[A-Za-z0-9_-]{1,64}$"
+      responses:
+        "202":
+          description: Wizard session created
+          content:
+            application/json:
+              schema:
+                type: object
+                required: [session_id, total, index]
+                properties:
+                  session_id: { type: string }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)